networkx/linalg/algebraicconnectivity.py

```diff
@@ -460,6 +460,17 @@ def fiedler_vector(
     See Also
     --------
     laplacian_matrix
+
+    Examples
+    --------
+    Given a connected graph the signs of the values in the Fiedler vector can be
+    used to partition the graph into two components.
+
+    >>> G = nx.cycle_graph(4)
+    >>> print(nx.fiedler_vector(G, normalized=True, seed = 1))
+    [-0.69141345 -0.1481467   0.69141344  0.14814671]
+
+    The connected components are {0,1} and {2,3}.
```

Comment:
```suggestion
    >>> G = nx.barbell_graph(5, 0)
    >>> nx.fiedler_vector(G, normalized=True, seed=1)
    array([-0.32864129, -0.32864129, -0.32864129, -0.32864129, -0.26072899,
            0.26072899,  0.32864129,  0.32864129,  0.32864129,  0.32864129])
    >>> nx.fiedler_vector(G, normalized=True, seed=1)
    array([-0.32864129, -0.32864129, -0.32864129, -0.32864129, -0.26072899,
            0.26072899,  0.32864129,  0.32864129,  0.32864129,  0.32864129])

    The connected components are the two 5-node cliques of the barbell graph.
```

Reply:
Yes, that's true. I also thought of the karate club graph but wanted a small example.  This is perfect!